docs/test-pr-guide.md

```diff
@@ -23,6 +23,40 @@ git pull https://github.com/<contributor>/mentorship-backend/ <branch-name>
 
 8. Get screenshots/gifs of before and after the fix and attach them to the PR comment.
 
+9. If the testing is unsuccessful and fixes are needed change then remove the label _Needs Testing_ and add _Needs Review_.
+
+10. If the testing is successful and no improvements needed then remove the label _Needs Testing_ and add _Ready to Merge_.
```

Comment:
Sugestion: If the testing is successful and no improvements needed, then remove the label `Needs Testing` and add label `Ready to Merge`